lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -48,8 +49,8 @@ class Inclusion:
                          'N': Inclusion.NOT,
                          'F': Inclusion.FILTER}
 
-INCLUDE_SECTION_PATTERN = re.compile('chapter|sequential')
-INCLUDE_PROBLEM_PATTERN = re.compile('problem')
+INCLUDE_SECTION_PATTERN = re.compile('|'.join(settings.INSTRUCTOR_QUERY_SECTION_TYPES))
```

Comment:
Let's not future-proof this just yet, which is to say,
let's undo this for now.

I'm not sure what else this would be; @dcadams ?